### ISSUE TITLE
Different Dimensions and a couple others mods ( suggestion )

### ISSUE THREAD

**Issue by DragonKing810** [kreezxil/World-of-Dragons#447]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]  **not really**

**Describe the solution you'd like**
A clear and concise description of what you want to happen. to be able to have diversity in multiple different Dimensions ( stackable ) / weaponry's / more enchantments

  

**Describe alternatives you've considered**
messed with it a little and was able to add these
https://www.curseforge.com/minecraft/mc-mods/dimension-stacking-rifts
https://www.curseforge.com/minecraft/mc-mods/the-beneath
https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest
https://www.curseforge.com/minecraft/mc-mods/the-erebus
https://www.curseforge.com/minecraft/mc-mods/aroma1997s-dimensional-world
https://www.curseforge.com/minecraft/mc-mods/dungeon-tactics
https://www.curseforge.com/minecraft/mc-mods/extended-days
https://www.curseforge.com/minecraft/mc-mods/so-many-enchantments
https://www.curseforge.com/minecraft/mc-mods/random-enchants
https://www.curseforge.com/minecraft/mc-mods/spartan-weaponry
https://www.curseforge.com/minecraft/mc-mods/spartan-and-fire
https://www.curseforge.com/minecraft/mc-mods/quality-tools
https://www.curseforge.com/minecraft/mc-mods/living-enchantment
https://www.curseforge.com/minecraft/mc-mods/enchantments-control
https://www.curseforge.com/minecraft/mc-mods/enchanting-plus

all these made it a bit harder but it was alot of fun still having fun with it xD hopefully some of these get added.

**Comment by AzureDoom** [comment]
JFC and we thought the modpack was bloated before...this would kill it.

**Comment by DragonKing810** [comment]
it didnt feel that bloated to me when i added them it gave me alot of options but it also made it hell for me but fun at the same time since enchantments you can even get curses applied onto you which can make it annoying but more cunning play style as for the Dimensions each came with it own challenges and adding the modpack's difficulty made it fun :P

**Comment by kreezxil** [comment]
Yeah you have a beastly rig too. Most minecraft modded gamers play on 4 to
8gb rigs. rarely 16gb, uniquely 32gb (i'm in this group, with a 32gb 8core
pre-ryzen system) and epicly on 64GB. WOD used to be uber popular at one
time and guess when that was, when it ran on a 4GB system. Right now it
runs with struggle on 8gb systems. These additions will remove it from the
8gb system users too.

I do thank you for suggestions. If desired I'll make a new version of WOD
called WOD - BLOATED EDITION. I'm not joking ask Azure Doom, hell just
search "kreezxil"on the launcher, I've got 10 variants of WOD to handle
nearly every desire, and if your desire is reminiscent of at least more
people than yourself it would be a worthwhile alternative branch to me.

**Comment by DragonKing810** [comment]
thanks for replying and i look forward to it if it gets made as i do play with 24-32gigs of memory with a ryzen 7 3800x with a Nvidia 2080 super

**Comment by kreezxil** [comment]
Just so you know. 64 bit java only allocates at max 16GB, this is not a
launcher limitation but a java limitation, at 16GB almost 2GB will vanish
to overhead and gc management. Minecraft clients also only use 1 thread
whereas servers will use all that you have. However, servers too are also
limited to 16GB. I think this is one of the reasons we see an ebb every
now and then for Mojang to retire Java Edition and go with C++ Edition but
open it to modding.

Ironically, that's called Bedrock Edition.

**Comment by DragonKing810** [comment]
i knew about that reason is cause i do other stuff + minecraft and minecraft eats like half my memory most of the time while i use my browser ( waiting for certain stuff to craft )

**Comment by hellwolf95** [comment]
well extended days screws with minecraft itself as you will notice a jaring sun movement. but if you wish to have a chunk of those mods why don't you try Realm of Mazara and have your PC explode.

**Comment by kreezxil** [comment]
Forgot about that. You should play ROM it massive in scope as she says with
those mods. I think it might have dragons too and if not you can add the
mod to it easily.
